ReactNativeTemplate/android/build.gradle

```diff
@@ -1,6 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+  ext {
+    buildToolsVersion = "28.0.3"
+    minSdkVersion = 21
+    compileSdkVersion = 28
+    targetSdkVersion = 28
+    supportLibVersion = "28.0.0"
```

Comment:
Shouldn't this be the same as `buildToolsVersion`?

Reply:
I don't think there is a 28.0.3 version of the support lib (see https://developer.android.com/topic/libraries/support-library/revisions).